docs/upload/overview.mdx

```diff
@@ -110,6 +110,7 @@ _An asterisk denotes that an option is required._
 | **`pasteURL`**                 | Controls whether files can be uploaded from remote URLs by pasting them into the Upload field. **Enabled by default.** Accepts `false` to disable or an object with an `allowList` of valid remote URLs. [More](#uploading-files-from-remote-urls) |
 | **`resizeOptions`**            | An object passed to the the Sharp image library to resize the uploaded file. [More](https://sharp.pixelplumbing.com/api-resize)                                                                                                                    |
 | **`skipSafeFetch`**            | Set to an `allowList` to skip the safe fetch check when fetching external files. Set to `true` to skip the safe fetch for all documents in this collection. Defaults to `false`.                                                                   |
+| **`allowRestrictedFileTypes`** | Set to `true` to allow restricted file types. Defaults to `false`. [More](#restricted-file-types)                                                                                                                                                  |
```

Comment:
I would mention in the description that this flag is ignored if `mimeTypes` are provided as you did in the types 👍